index.d.ts

```diff
@@ -1,426 +1,339 @@
+    errorOnAbortedRequests?: boolean;
+    filterHttpHeaders?: boolean;
+    frameworkName?: string;
+    frameworkVersion?: string;
+    globalLabels?: KeyValueConfig;
+    hostname?: string;
+    ignoreMessageQueues?: Array<string>;
+    ignoreUrls?: Array<string | RegExp>;
+    ignoreUserAgents?: Array<string | RegExp>;
+    instrument?: boolean;
+    instrumentIncomingHTTPRequests?: boolean;
+    kubernetesNamespace?: string;
+    kubernetesNodeName?: string;
+    kubernetesPodName?: string;
+    kubernetesPodUID?: string;
+    logLevel?: LogLevel;
+    logUncaughtExceptions?: boolean;
+    logger?: PinoLogger | Logger;
+    maxQueueSize?: number;
+    metricsInterval?: string; // Also support `number`, but as we're removing this functionality soon, there's no need to advertise it
+    metricsLimit?: number;
+    payloadLogFile?: string;
+    centralConfig?: boolean;
+    sanitizeFieldNames?: Array<string>;
+    secretToken?: string;
+    serverCaCertFile?: string;
+    serverTimeout?: string; // Also support `number`, but as we're removing this functionality soon, there's no need to advertise it
+    serverUrl?: string;
+    serviceName?: string;
+    serviceNodeName?: string;
+    serviceVersion?: string;
+    sourceLinesErrorAppFrames?: number;
+    sourceLinesErrorLibraryFrames?: number;
+    sourceLinesSpanAppFrames?: number;
+    sourceLinesSpanLibraryFrames?: number;
+    spanFramesMinDuration?: string;
+    stackTraceLimit?: number;
+    transactionIgnoreUrls?: Array<string>;
+    transactionMaxSpans?: number;
+    transactionSampleRate?: number;
+    useElasticTraceparentHeader?: boolean;
+    usePathAsTransactionName?: boolean;
+    verifyServerCert?: boolean;
+  }
 
-interface Taggable {
-  setLabel (name: string, value: LabelValue, stringify?: boolean): boolean;
-  addLabels (labels: Labels, stringify?: boolean): boolean;
-}
+  interface CaptureErrorOptions {
+    request?: IncomingMessage;
+    response?: ServerResponse;
+    timestamp?: number;
+    handled?: boolean;
+    user?: UserObject;
+    labels?: Labels;
+    tags?: Labels;
+    custom?: object;
+    message?: string;
+    captureAttributes?: boolean;
+    skipOutcome?: boolean;
+  }
 
-interface StartSpanFn {
-  startSpan(
-    name?: string | null,
-    options?: SpanOptions
-  ): Span | null;
-  startSpan(
-    name: string | null,
-    type: string | null,
-    options?: SpanOptions
-  ): Span | null;
-  startSpan(
-    name: string | null,
-    type: string | null,
-    subtype: string | null,
-    options?: SpanOptions
-  ): Span | null;
-  startSpan(
-    name: string | null,
-    type: string | null,
-    subtype: string | null,
-    action: string | null,
-    options?: SpanOptions
-  ): Span | null;
-}
+  interface Labels {
+    [key: string]: LabelValue;
+  }
 
-// Inlined from @types/aws-lambda - start
-declare namespace AwsLambda {
```

Comment:
REVIEWER NOTE: I don't see the value in inlining these rather than include "@types/aws-lambda" in devDependencies (as we do for node and pino types).